src/gfn/utils/handlers.py

```diff
@@ -40,3 +43,16 @@ def is_callable_exception_handler(
             f"conditioning was passed but {target_name} is not callable: {type(target)}"
         )
         raise
+
+
+def warn_about_recalculating_logprobs(
```

Comment:
This is a very useful warning. I'll be a bit pedantic here:
Sometimes, log_probs are stored, but they are not the log_probs that should be used for the loss (e.g. the training objects come from another gflownet). Can you make if more explicit, by saying. "If this is the case, you should instead ..."

Reply:
Pedantic is good! thanks!